src/migrations.js

```diff
@@ -0,0 +1,61 @@
+import walkSync from 'walk-sync'
+import { readFileSync } from 'fs'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['readFileSync' is defined but never used.](https://app.codacy.com/manual/trompamusic/ce-api/pullRequest?prid=5244919)